libvast/src/plugin.cpp

```diff
@@ -20,6 +20,7 @@
 #include "vast/error.hpp"
 #include "vast/ids.hpp"
 #include "vast/logger.hpp"
+#include "vast/pipeline_operator.hpp"
```

Comment:
Why do we need this include here?

Reply:
Removed in https://github.com/tenzir/vast/pull/2877/commits/d20eb9780a49408de5cec281a1ddd8f3f2e538b7